### PR TITLE
#200 - set columns width in benefits Excel report

### LINKED ISSUE #200: Set columns width in benefits Excel report
Report for benefits look quite good:

image

But if the user downloads the report in .xlsx file, he sees this:

image

Columns should be narrowed and text in table headers should be wrapped. It will improve this file readability.

### PR DESCRIPTION
**Pull request by dawidrudnik**
It should close #200

### CODE REVIEW

File: app/Domain/BenefitsReportTimesheet.php [hunk 1]
```diff
@@ -122,5 +125,16 @@ public function styles(Worksheet $sheet): void
         $sheet->getStyle("B2:{$lastColumn}{$lastRow}")
             ->getNumberFormat()
             ->setFormatCode(NumberFormat::FORMAT_NUMBER_00);
+
+        for ($col = 1; $col <= Coordinate::columnIndexFromString($lastColumn); $col++) {
+            $sheet->getColumnDimensionByColumn($col)->setWidth(22);
+        }
```
Comment (krzysztofrewak) [inline app/Domain/BenefitsReportTimesheet.php:131]:
```suggestion
        for ($column = 1; $column <= Coordinate::columnIndexFromString($lastColumn); $column++) {
            $sheet->getColumnDimensionByColumn($column)->setWidth(22);
        }
```

File: app/Domain/BenefitsReportTimesheet.php [hunk 2]
```diff
@@ -122,5 +125,16 @@ public function styles(Worksheet $sheet): void
...
+
+        $sheet->getRowDimension(1)
+            ->setRowHeight(70);
```
Comment (krzysztofrewak) [inline app/Domain/BenefitsReportTimesheet.php:134]: Move this to named constant, please.

File: resources/js/Shared/Widgets/RemoteWorkList.vue [hunk 3]
```diff
@@ -71,7 +71,7 @@
             <template #head>
               <HomeCityIcon
                 class="flex justify-center"
-                size="48"
+                :size="48"
               />
```
Comment (krzysztofrewak) [inline resources/js/Shared/Widgets/RemoteWorkList.vue:75]: Why not inline?
```html
<HomeCityIcon class="flex justify-center" :size="48"/>
```

Or even:
```html

```html
<HomeCityIcon class="flex justify-center" :size="48"></HomeCityIcon>
```
```

File: app/Domain/BenefitsReportTimesheet.php [hunk 4]
```diff
@@ -46,7 +46,10 @@ public function generator(): Generator
             return [
                 "user" => $user["name"],
                 "benefits" => Arr::map($item["benefits"], function ($benefit): array {
```
Comment (krzysztofrewak) [inline app/Domain/BenefitsReportTimesheet.php:48]: Please type `$benefit` here.

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]

File: resources/js/Shared/Widgets/RemoteWorkList.vue (hunk 3 above)
Comment (dawidrudnik) [reply resources/js/Shared/Widgets/RemoteWorkList.vue:75]: We enforce limits to the one number of attributes/properties per line. That's the `vue/max-attributes-per-line` rule.

Comment (krzysztofrewak) [reply resources/js/Shared/Widgets/RemoteWorkList.vue:75]: I don't like it, but okay.

**Review by krzysztofrewak — APPROVED** [review]

**Review by EwelinaLasowy — APPROVED** [review]

**Pull request merged**